webserver/src/main/java/io/kestra/webserver/controllers/ExecutionController.java

```diff
@@ -145,26 +146,47 @@ public PagedResults<Execution> find(
         @Parameter(description = "A flow id filter") @Nullable @QueryValue String flowId,
         @Parameter(description = "The start datetime") @Nullable @Format("yyyy-MM-dd'T'HH:mm[:ss][.SSS][XXX]") @QueryValue ZonedDateTime startDate,
         @Parameter(description = "The end datetime") @Nullable @Format("yyyy-MM-dd'T'HH:mm[:ss][.SSS][XXX]") @QueryValue ZonedDateTime endDate,
+        @Parameter(description = "A start time range filter relative to the current time", examples = {
+            @ExampleObject(name = "Filter last 5 minutes", value = "PT5M"),
+            @ExampleObject(name = "Filter last 24 hours", value = "P1D")
+        }) @Nullable @QueryValue Duration startDateRange,
+        @Parameter(description = "An end time range filter relative to the current time", examples = {
+            @ExampleObject(name = "Filter up to 5 minutes", value = "PT5M"),
+            @ExampleObject(name = "Filter up to 24 hours", value = "P1D")
+        }) @Nullable @QueryValue Duration endDateRange,
         @Parameter(description = "A state filter") @Nullable @QueryValue List<State.Type> state,
         @Parameter(description = "A labels filter as a list of 'key:value'") @Nullable @QueryValue List<String> labels,
         @Parameter(description = "The trigger execution id") @Nullable @QueryValue String triggerExecutionId,
         @Parameter(description = "A execution child filter") @Nullable @QueryValue ExecutionRepositoryInterface.ChildFilter childFilter
     ) {
+        final ZonedDateTime now = ZonedDateTime.now();
+
         return PagedResults.of(executionRepository.find(
             PageableUtils.from(page, size, sort, executionRepository.sortMapping()),
             query,
             tenantService.resolveTenant(),
             namespace,
             flowId,
-            startDate,
-            endDate,
+            resolveAbsoluteDateTime(startDate, startDateRange, now),
+            resolveAbsoluteDateTime(endDate, endDateRange, now),
             state,
             RequestUtils.toMap(labels),
             triggerExecutionId,
             childFilter
         ));
     }
 
+    private ZonedDateTime resolveAbsoluteDateTime(ZonedDateTime absoluteDateTime, Duration timeRange, ZonedDateTime now) {
```

Comment:
Hi @yuri1969, thank you so much for your pull request. Could you please write some unit tests with one or two cases for that method. For doing that, you can remove the `private` access specifier (i.e, no modifier specified) and annotate the method with `@VisibleForTesting`. Thank you.

Reply:
Done